AI-and-Analytics/Getting-Started-Samples/IntelPython_daal4py_GettingStarted/README.md

```diff
@@ -151,4 +154,24 @@ Here is one of our loaded model's features:
    1.58423529e-02 -4.57542900e-01]]
 [CODE_SAMPLE_COMPLETED_SUCCESFULLY]
 ```
-
+### Running Samples on the Intel&reg; DevCloud (Optional)<a name="run-samples-on-devcloud"></a>
+
+<!---Include the next paragraph ONLY if the sample runs in batch mode-->
+### Run in Batch Mode
+This sample runs in batch mode, so you must have a script for batch processing. Once you have a script set up, refer to [Running the Sample](#running-the-sample).
```

Comment:
Should refer to Running the Sample as a Python File earlier in the README (after Jupyter instructions)

Reply:
@racheloberman-intel I just pushed a new commit. Let me know if that looks correct.
I moved the node information up along with the Run in Batch Mode information because the node has been a problem for users trying to connect to DevCloud.